packages/@sanity/desk-tool/src/components/paneItem/ReferencedDocHeading.tsx

```diff
@@ -0,0 +1,74 @@
+import React, {ReactNode} from 'react'
+import {Popover, Box, Text, Heading, Flex, Button, Stack, ThemeColorProvider} from '@sanity/ui'
+import {LinkIcon} from '@sanity/icons'
+import {TitleText, LinkCircle, HorizontalLine, StyledBox} from '../pane/PaneHeader.styles'
+import {useDeskToolSetting} from '../../settings'
+import {ReferencedDocTooltip} from './ReferencedDocTooltip'
+
+export function ReferencedDocHeading({title}: {title?: ReactNode}) {
+  const [hidePopover, setHidePopover] = useDeskToolSetting(
+    'desk-tool',
+    'referenced-doc-has-confirmed-dialog',
+    false
+  )
+
+  if (hidePopover) {
+    return (
+      <Flex>
+        <TitleText tabIndex={0} textOverflow="ellipsis" weight="semibold">
+          {title}
+        </TitleText>
+        <ReferencedDocTooltip />
+      </Flex>
+    )
+  }
+
+  const onClose = () => {
+    setHidePopover(true)
+  }
+
+  return (
+    <ThemeColorProvider tone="default">
+      <Box marginLeft={2}>
+        <Popover
+          content={<InnerPopover onClose={onClose} />}
+          placement="bottom-start"
+          referenceElement={document.getElementById('referenceTitle')}
```

Comment:
This should use a reference (`useRef`) - multiple document panes can be open at the same time, which will create multiple elements with the same ID